seclea_ai/svc/api/collection/aysnc_dataset.py

```diff
@@ -0,0 +1,60 @@
+import json
+import os
+import aiohttp
+
+from seclea_ai.lib.seclea_utils.core.transmission import Transmission
+
+root = "/collection/datasets"
+
+async def async_post_dataset(
+    transmission: Transmission,
+    dataset_file_path: str,
+    project_pk: str,
+    organization_pk: str,
+    name: str,
+    metadata: dict,
+    dataset_hash: str,
+    parent_dataset_hash: str = None,
+    delete=False,
+):
+    """
+    @param transmission:
+    @param dataset_file_path:
+    @param project_pk:
+    @param organization_pk:
+    @param name:
+    @param metadata:
+    @param dataset_hash:
+    @param parent_dataset_hash:
+    @param delete: delete dataset file
+    @return:
+    """
+    dataset_queryparams = {
+        "project": str(project_pk),
+        "organization": str(organization_pk),
+        "name": name,
+        "metadata": json.dumps(metadata),
+        "hash": str(dataset_hash),
+    }
+
+    if parent_dataset_hash and parent_dataset_hash != None:
```

Comment:
```suggestion
    if parent_dataset_hash is not None:
```
The if parent_dataset_hash is redundant it will either be a passed in argument or None. 

Also this is not the pythonic way to check for not None - the suggestion is the correct way.